litellm/main.py

```diff
@@ -5,10 +5,10 @@
 # |                                               |
 # +-----------------------------------------------+
 #
-#  Thank you ! We ❤️ you! - Krrish & Ishaan
```

Comment:
there seems to be too large a diff - it looks like your dev environment made some formatting changes

Reply:
ouchhh true... let me fix it, so sorry!